efficientdet/utils.py

```diff
@@ -296,8 +275,9 @@ def call(self, inputs, training=None):
 def batch_norm_class(is_training, strategy=None):
   if is_training and strategy == 'tpu':
     return TpuBatchNormalization
-  elif is_training and strategy == 'horovod':
-    return SyncBatchNormalization
+  elif is_training and strategy == 'gpus':
+    # TODO(fsx950223): Wait for #41980 to be fixed
```

Comment:
What does this mean?

Reply:
Tensorflow issue #41980.
It's a bug in keras.syncbatchnormalization which is caused by incorrect implementation of nccl all_reduce

Reply:
Could you replace it with something like:

```
# TODO(fsx950223): use SyncBatchNorm after the TF bug (incorrect nccl all_reduce) is fixed:
#  see https://github.com/tensorflow/tensorflow/issues/41980
```